### ISSUE TITLE
css files refer to themselves

### ISSUE THREAD

**Issue by mdorman** [perl-pod/pod-simple#89]
If you look at https://github.com/perl-pod/pod-simple/blob/master/lib/Pod/Simple/HTMLBatch.pm#L729-L738, it is pretty obvious that half the .css files are going to end up referring to _themselves_, rather than the files they're supposed to be overriding---`$outname` and `$variation` never vary.

Given that at least on my system, one of these "I import myself" files is the default, and the whole business becomes kind of useless.

**Comment by khwilliamson** [comment]
I'm getting ready to release a new version of all of Pod::Simple, and would like to fix this in it.  But I don't know anything about this area of the code, and it is not obvious to me what the problem here is.  You could save me a bunch of time if you could briefly give me an overview of what is here going on here.   I know what a css file is, but I don't know why they are being generated here, and apparently one for each color combination.  It just doesn't make sense for me without some context.

**Comment by theory** [comment]
It looks like each of the variations, `blkbluw`, `whtpurk`, `whtgrng`, and `grygrnw`, gets written to a file that just imports itself. IOW, the contents of `_ blkbluw.css` would be:

    /* This file is autogenerated.  Do not edit.  blkbluw */
    
    @import url("./_blkbluw.css");
    .indexgroup { display: none; }
    
    
Which seems less than useful. What is the purpose of these CSS files?

**Comment by theory** [comment]
I suspect it should do something like this:

``` perl
for (my ($outfile, $variation) = each %{{
    blkbluw => 'black_with_blue_on_white',
    whtpurk => 'white_with_purple_on_black',
    whtgrng => 'white_with_green_on_grey',
    grygrnw => 'grey_with_green_on_white',
}}) {
```

**Comment by khwilliamson** [comment]
This should now be fixed in version 3.36 now available from CPAN
